### ISSUE TITLE
Test Privacy Settings Page Interactive Elements for Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4913]
# Page to Test:

Privacy Settings - https://openlibrary.org/account/privacy

# Steps

Steps for this task are outlined in #4897
 
# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by SaravgiYash** [comment]
@bpmcneilly I would like to work on this issue.

**Comment by SaravgiYash** [comment]
@bpmcneilly No focus on Save Button due to `outline: 0;` 
Should I create a separate PR for all the minor HTML, CSS changes so that it can get merged?
For e.g: Fix order of Header HTML, Button outline CSS, and Search by focus?

**Comment by tuminzee** [comment]
@Yashs911 are you still working on this issue?

**Comment by shaiksahilrizwan** [comment]
@RayBB  I'm here to take this one on! Please assign it to me and I will start testing the Privacy Settings page .

**Comment by RayBB** [comment]
@shaiksahilrizwan assigned

**Comment by shaiksahilrizwan** [comment]
Hi @RayBB, 

I have completed the manual keyboard testing (Tab / Shift+Tab) for the **Privacy Settings** page. 

**Findings:**
* All form fields, checkboxes, and buttons in the main content area receive keyboard focus in a logical order.
* Focus indicators are clearly visible on all interactive elements.
* WAVE Tool check (Scored 9.8/10) which returned 0 errors—just a few minor alerts like redundant text which are outside the scope of this keyboard accessibility epic .

Since the keyboard accessibility on this page is working perfectly and there are no bugs to file, I believe we can close this issue! and if there are any others from the epic you'd like me to tackle next.

**Comment by RayBB** [comment]
@shaiksahilrizwan thank you so much for doing a through test and reporting your findings here. This is excellent work and it's really important that we're getting through these. Now we just have a handful left from the main issue.

This next one is a little more tricky but if you're interested to take it on that would be awesome 
https://github.com/internetarchive/openlibrary/issues/4911
